### DISCUSSION TITLE
Toggle Terminus window should do autofocus

### DISCUSSION THREAD

**Discussion by ignoch** [Eugeny/tabby#3744] [Q&A]
I've using Terminus for quite a while because of the fact that uses the toggle invocation window, which I love, but when I do the toggle hotkey it loses focus and I need to use the mouse to regain it.  It is a way to regain the focus when toggling the console? probably a setting that I can't find it?
Keep the good work.

**Comment by Eugeny** [comment]
It should already auto-focus - which platform are you on?
